x/merkledb/db.go

```diff
@@ -125,8 +126,8 @@ type Config struct {
 	// RootGenConcurrency is the number of goroutines to use when
 	// generating a new state root.
 	//
-	// If 0 is specified, [runtime.NumCPU] will be used. If -1 is specified,
-	// no limit will be used.
+	// If 0 is specified, [runtime.NumCPU] will be used.
+	// Must be non-negative.
 	RootGenConcurrency int
```

Comment:
should make uint if must be positive

Reply:
Yeah I noticed the same. There are a bunch of other config fields that should uints. I will make a follow up PR to change them.

Reply:
Ill change this to a uint though